jasmine.js

```diff
@@ -0,0 +1,41 @@
+describe("array numbers", function() {
+
+	it("check elements from array", function() {
+	 	expect(tests.numbers).toContain(1);
+    		expect(tests.numbers).not.toContain("1");
+    		expect(tests.numbers).not.toContain("abcd");
+  	});
```

Comment:
I refactored this part of the test to. I just want to make sure that  the array has only numbers.